python/py-pyobjc6/Portfile

```diff
@@ -45,6 +53,26 @@ if {${name} ne ${subport}} {
                     patch-docs-conf.py.diff \
                     patch-install.py.diff
 
+    # Backport newer code from upstream:
+    patchfiles-append \
+                    0001-Fix-obsolete-Python-API.patch
```

Comment:
This is a nearly 4000-line patch. I'm never thrilled about patches that huge since they're unmaintainable. But I guess this port is deliberately at a much older version to support older systems and needs this patch to build with newer python, so the port will never be updated to a newer version so no maintenance of the patch would be necessary. You said the patch comes from upstream. Could you add the upstream URL?

Reply:
@ryandesign Yes, it will not be updated at all, since the upstream did not express an interest, and neither do I.
Having the port is still useful though, since some software may be able to build against it.

As I recall, updates were just dumped as a single commit, not commit-per-change, but I will find where it was introduced, once back to the laptop.

Reply:
@ryandesign I have updated comments with links and a notice update [a lack of] future version updates. I also claim maintainership to make it clear that I take care of patches, be it necessary.